app/src/main/java/com/example/bigowlapp/utils/PeriodicLocationCheckAlarmManager.java

```diff
@@ -8,6 +8,7 @@
 import android.content.Intent;
 import android.content.pm.PackageManager;
 import android.os.SystemClock;
+import android.widget.Toast;
```

Comment:
Remove this useless import and empty line 58

Reply:
ok done